src/solvers/dg/3d/amr.jl

```diff
@@ -396,6 +396,70 @@ function calc_amr_indicator(dg::Dg3D, mesh::TreeMesh, time::Float64)
         lambda[element_id] = 0.0
       end
     end
+  elseif dg.amr_indicator === :blob                                                                 
+    base_level = 4                                                                                  
+    max_level = 7                                                                                   
+    super_max_level = 7                                                                             
```

Comment:
`max_level == super_max_level` - is this intentional?

Reply:
yes!
the blob needs a super max level...when we get MPI, we will increase this to 8 😎